### ISSUE TITLE
Celebrity Faces Detection using Image Processing

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#461]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Celebrity Faces Detection using Image Processing</br>
:red_circle: **Aim** : The aim of this project is to detect the faces of the celebs based on the dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/jessicali9530/celeba-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Avdhesh-Varshney** [comment]
@abhisheks008 can you assign me this issue to work under SWOC S4?

- Full name : Avdhesh Varshney
- GitHub Profile Link : https://github.com/Avdhesh-Varshney
- Email ID : avdheshvarshney786@gmail.com
- Participant ID (if applicable): SWOC'24 Contributor
- Approach for this Project : 
  - Analyzing the dataset.
  - EDA processing.
  - Filtering the dataset or performing feature engineering so that which can easily fit on the model.
  - Segregating the training set of images and testing images dataset.
  - Creating deep neural networks model, and cnn models for classification of different faces.
  - Tune the hyper-parameters and improving the loss and accuracy of the model.
  - Trying to apply any other algorithm to classify the images and get learning and insights.
  - Summarizing all the algorithm's efficiency and scores at the end of the notebook.
  - Model comparison with their precision, f1 score and recall scores.
  - Identifying the best fit algorithm and plot the graph of that algorithm as how it's fit it on training set and test set.
- What is your participant role? SWOC'24

**Comment by abhisheks008** [comment]
Issue assigned to @Avdhesh-Varshney under SWOC S4.
